unit_tests/src/test_rc_data_obj.cpp

```diff
@@ -1528,3 +1530,23 @@ TEST_CASE("#7250: updates to ticket stats do not hang")
         }
     }
 }
+
+TEST_CASE("api plugin")
+{
+    load_client_api_plugins();
+
+    irods::experimental::client_connection conn{irods::experimental::defer_authentication};
+
+    DataObjInp input{};
+    char* json_error_string{};
+
+    CHECK(rc_register_physical_path(static_cast<RcComm*>(conn), &input, &json_error_string) == SYS_NO_API_PRIV);
+    CHECK(json_error_string == nullptr);
+}
+
+TEST_CASE("api plugin")
+{
+    load_client_api_plugins();
+    irods::experimental::client_connection conn{irods::experimental::defer_authentication};
+    CHECK(rc_touch(static_cast<RcComm*>(conn), "") == SYS_NO_API_PRIV);
+}
```

Comment:
How do we feel about these test cases living here?

NGL, I really didn't want to create two additional files to test these cases. If others feel that is incorrect, then I'll create the files.

Thoughts?

Reply:
This file is becoming a bit of a dumping ground anyway, so it seems fine here to me. I would request that the test case names be a little more descriptive and/or a comment indicating the purpose, please :)

Reply:
Agreed. Those names are placeholders until I figure out how to describe what they're about.

Reply:
I still have to write an issue for this PR too.